demosplan/DemosPlanCoreBundle/Permissions/Permissions.php

```diff
@@ -986,8 +957,6 @@ public function checkProcedurePermission(): void
     {
         // Prüfe, ob der User ins Verfahren darf
         if (null !== $this->procedure) {
-            $this->setProcedurePermissions();
```

Comment:
Shouldn't it be called somewhere else?
This may explain why access to a procedure isn't possible anymore?
(Admin hasn't the "area_admin_dashboard" permission enabled)

Reply:
this was also accidentally pushed to #2991 thank you. reverted in [4483601](https://github.com/demos-europe/demosplan-core/pull/2986/commits/4483601256e62c20151af602c2ec269f5521ac50) but I wonder why this branch worked locally